### PR TITLE
Make Cancellable Send+Sync and don't generate trait for it

### PR DESCRIPTION
**Pull request by sdroege**
It can't be subclassed.

### CODE REVIEW

File: src/auto/versions.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
 All files in this directory
 were generated by gir (https://github.com/gtk-rs/gir @ 6855214)
-from gir-files (https://github.com/gtk-rs/gir-files @ cf55bdc)
+from gir-files (https://github.com/gtk-rs/gir-files @ db49619)
```
Comment (EPashkin) [inline src/auto/versions.txt]: db49619 is too old

**Comment by EPashkin** [comment]
👍 without `version.txt` change

File: src/auto/versions.txt (hunk 1 above)
Comment (sdroege) [reply src/auto/versions.txt]: Yes, fixed

**Comment by sdroege** [comment]
Without this, `Cancellable` is basically useless as you would use it to cancel something that blocks your current thread usually :)

**Comment by sdroege** [comment]
Should be good to go then

**Comment by tmiasko** [comment]
If you don't need `reset` I would drop it from bindings.

**Comment by sdroege** [comment]
Ugly indeed :) I send another PR for that later

**Comment by GuillaumeGomez** [comment]
Code looks good to me. Should I wait for the other PR before merging this one?

**Comment by sdroege** [comment]
They're independent things

**Pull request merged**